surface/include/pcl/surface/impl/marching_cubes_hoppe.hpp

```diff
@@ -60,26 +60,46 @@ pcl::MarchingCubesHoppe<PointNT>::~MarchingCubesHoppe ()
 template <typename PointNT> void
 pcl::MarchingCubesHoppe<PointNT>::voxelizeData ()
 {
+  const bool is_far_ignored = dist_ignore_ > 0.0f;
+
+  std::vector<int> nn_indices(1, 0);
+  std::vector<float> nn_sqr_dists(1, 0.0f);
+  Eigen::Vector3f point;
+  PointNT p;
```

Comment:
Comment regarding the two lines above: I'm an apologist of RAII. Does this grant you a super duper performance increase to justify declaring it here?